test/onigumo_cli_test.exs

```diff
@@ -13,11 +13,22 @@ defmodule OnigumoCLITest do
     "-c"
   ]
 
+  @invalid_combinations [
+    "--help -C invalid",
+    "-h --invalid",
+    "downloader -h"
+  ]
```

Comment:
```suggestion
  @invalid_combinations [
    ["--help", "-C", "invalid"],
    ["-h", "--invalid"],
    ["downloader", "-h"]
  ]
```

`argv` is passed as a list of arguments, not as a space-separated string. As a result, you will see the `"downloader -h"` test failing. See my comments in https://github.com/Glutexo/onigumo/pull/238#issuecomment-2308952126.